L9-go-resolver-balancer-API.md

```diff
@@ -53,7 +53,7 @@ gRPC first gets an address picked by the balancer. Balancer picks the address ba
 
 #### The balancing policy needs to be changed on service config updates 
 
-Upon service config updating, gRPC will look at the LoadBalancingPolicy field and decide which balancing policy to use. If the LoadBalancingPolicy specified by the service config is different from what's currently being used, gRPC should be able to change the policy. Refer to go/grpc-service-config for more details on this.
+Upon service config updating, gRPC will look at the LoadBalancingPolicy field and decide which balancing policy to use. If the LoadBalancingPolicy specified by the service config is different from what's currently being used, gRPC should be able to change the policy.
```

Comment:
You could otherwise replace it with a link to https://grpc.io/docs/guides/service-config/.

Reply:
Updated